data/data-pipeline/data_pipeline/etl/sources/census_decennial/etl.py

```diff
@@ -147,6 +148,65 @@ def __init__(self):
             field_names.CENSUS_DECENNIAL_UNEMPLOYMENT_FIELD_2009
         )
 
+        # Race/Ethnicity fields
+        self.TOTAL_RACE_POPULATION_FIELD = "PCT086001"  # Total
+        self.ASIAN_FIELD = "PCT086002"  # Total!!Asian
+        self.BLACK_OR_AA_FIELD = "PCT086003"  # Total!!Black or African American
+        self.NATIVE_HI_OR_API_FIELD = (
+            "PCT086004"  # Total!!Native Hawaiian and Other Pacific Islander
+        )
+        self.WHITE_FIELD = "PCT086005"  # Total!!White
+        self.HISPANIC_OR_LATINO_FIELD = "PCT086006"  # Total!!Hispanic or Latino
+        self.TWO_OR_MORE_RACES_FIELD = (
+            "P004024"  # Total!!Two or More Ethnic Origins or RaceTotal
```

Comment:
Hmm. If I'm reading the [API docs](https://api.census.gov/data/2010/dec/pl/variables.html) right, this is for only 18 or over, right? 

Also I think it's "not hispanic" / "two or more", whereas the ACS one is just "two or more", so maybe we shouldn't make this comparison? This would leave "two or more" as null for the entire island areas but I think that's fine?

Reply:
Good catch --- I think I get excited about finding a two or more. I hadn't noticed that it was 18 or over so I am glad you did.

Reply:
Removed in b741001be6e1918273ddbc77f4431d507011d5df